bcda/cli_test.go

```diff
@@ -106,6 +106,86 @@ func (s *CLITestSuite) TestCreateACO() {
 	buf.Reset()
 }
 
+func (s *CLITestSuite) TestImportCCLF0() {
+	assert := assert.New(s.T())
+
+	cclf0filePath := "../shared_files/cclf/T.A0001.ACO.ZC0Y18.D181120.T1000011"
+	cclf0metadata, err := getCCLFFileMetadata(cclf0filePath)
```

Comment:
I'd recommend creating the `cclfFileMetadata` struct directly here rather than using `getCCLFFileMetadata()` to generate it. If something breaks in that function, it should break only the tests written specifically for it.

Reply:
yessss 👍